backend/OR-Tools/OR-Tools/Solvers/OrToolsConverter.cs

```diff
@@ -0,0 +1,245 @@
+                Totals = GetTotals(input, manager, routing, solution)
+            };
+        }
+
+        private IList<Dropped> GetDroppedLocations(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
+        {
+            IDictionary<int, string> locationsNames = input.Locations.ToDictionary(k => k.Id - 1, v => v.Name);
+            List<Dropped> droppedLocations = new List<Dropped>();
+            for (int index = 0; index < routing.Size(); ++index)
+            {
+                if (routing.IsStart(index) || routing.IsEnd(index))
+                {
+                    continue;
+                }
+
+                if (solution.Value(routing.NextVar(index)) == index)
+                {
+                    int node = manager.IndexToNode(index);
+                    droppedLocations.Add(new Dropped { LocationName = locationsNames[node] });
+                }
+            }
+            return droppedLocations;
+        }
+
+        private IList<Itineraries> GetItineraries(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
+        {
+            int vehiclesAmount = input.Vehicles.Count();
+            DateTime minDate = input.Locations.Select(l => l.From).Min();
+            IDictionary<int, string> vehiclesNames = input.Vehicles.ToDictionary(k => k.Id - 1, v => v.Name);
+
+            List<Itineraries> itineraries = new List<Itineraries>();
+
+            RoutingDimension capacityDimension = routing.GetDimensionOrDie("Capacity");
+            RoutingDimension timeDimension = routing.GetMutableDimension("Time");
+
+            for (int vehicleIndex = 0; vehicleIndex < vehiclesAmount; ++vehicleIndex)
+            {
+                long load = 0;
+                long routeIndex = routing.Start(vehicleIndex);
+
+                while (routing.IsEnd(routeIndex) == false)
+                {
+                    long previousRouteIndex = routeIndex;
+
+                    load += solution.Value(capacityDimension.CumulVar(routeIndex));
+
+                    long distance = routing.GetArcCostForVehicle(previousRouteIndex, routeIndex, 0);
+                    IntVar timeVar = timeDimension.CumulVar(routeIndex);
+
+                    itineraries.Add(new Itineraries
+                    {
+                        VehicleName = vehiclesNames[vehicleIndex],
+                        Load = (int)load,
+                        Distance = (int)distance,
+                        From = minDate.AddMinutes(solution.Min(timeVar)),
+                        To = minDate.AddMinutes(solution.Max(timeVar))
+                    });
+
+                    routeIndex = solution.Value(routing.NextVar(routeIndex));
+                }
+            }
+
+            return itineraries;
+        }
+
+        private IList<Summary> GetSummaries(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'manager'.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)